### ISSUE TITLE
Failed to activate the remote-edit-ni package

### ISSUE THREAD

**Issue by jonathankoren** [newinnovations/remote-edit-ni#4]
[Enter steps to reproduce:]

1. Install package via Atom's package tool.

**Atom**: 1.25.0 x64
**Electron**: 1.7.11
**OS**: Mac OS X 10.13.3
**Thrown From**: [remote-edit-ni](https://github.com/newinnovations/remote-edit-ni) package 3.0.0

### Stack Trace

Failed to activate the remote-edit-ni package

```
At Path must be a string. Received undefined

TypeError: Path must be a string. Received undefined
    at assertPath (path.js:7:11)
    at Object.resolve (path.js:1146:7)
    at /packages/remote-edit-ni/lib/model/remote-edit-editor.js:17:59)
    at /packages/remote-edit-ni/lib/model/remote-edit-editor.js:297:3)
    at Module.get_Module._compile (/Applications/Atom.app/Contents/Resources/app/src/native-compile-cache.js:106:36)
    at Object.value [as .js] (/Applications/Atom.app/Contents/Resources/app/src/compile-cache.js:240:29)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (/app.asar/static/index.js:47:45)
    at require (/Applications/Atom.app/Contents/Resources/app/src/native-compile-cache.js:66:33)
    at /packages/remote-edit-ni/lib/main.coffee:3:20)
    at /packages/remote-edit-ni/lib/main.coffee:1:1)
    at /packages/remote-edit-ni/lib/main.coffee:1:1)
    at Module.get_Module._compile (/Applications/Atom.app/Contents/Resources/app/src/native-compile-cache.js:106:36)
    at Object.value [as .coffee] (/Applications/Atom.app/Contents/Resources/app/src/compile-cache.js:240:29)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (/app.asar/static/index.js:47:45)
    at require (internal/module.js:20:19)
    at customRequire (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:96:26)
    at Package.requireMainModule (/Applications/Atom.app/Contents/Resources/app/src/package.js:782:33)
    at Package.activateNow (/Applications/Atom.app/Contents/Resources/app/src/package.js:220:40)
    at measure (/Applications/Atom.app/Contents/Resources/app/src/package.js:206:33)
    at Package.measure (/Applications/Atom.app/Contents/Resources/app/src/package.js:88:25)
    at activationPromise.Promise (/Applications/Atom.app/Contents/Resources/app/src/package.js:200:20)
    at Promise (<anonymous>)
    at Package.activate (/Applications/Atom.app/Contents/Resources/app/src/package.js:198:38)
    at PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app/src/package-manager.js:695:42)
```

### Commands

```
  2x -0:32.2.0 core:backspace (input.hidden-input)
```

### Non-Core Packages

```
minimap-git-diff 4.3.1 
remote-edit-ni 3.0.0 
split-diff 1.5.2 
```

**Comment by urban-1** [comment]
From the documentation, seems like `resourcePath` has been moved out of `atom.config` and into `atom` (the AtomEnvironment itself). I will see if I can make a quick fix/PR today but I am running atom 1.23.3 so I will need some time. If this is urgent you can:

1. Downgrade to 1.23.3
2. Test my theory by replacing 
    ```
    const {
        resourcePath
    } = atom.config;
    ```
    with 
    
    ```
    const {
        resourcePath
    } = atom;
    ```
    in `/packages/remote-edit-ni/lib/model/remote-edit-editor.js` lines 9-11

I will post any updates here

**Comment by newinnovations** [comment]
Merged patch by urban-1 and released on atom.io.
